l10n_it_vat_registries/report/report_registro_iva.xml

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="report_registro_iva">
+    <t t-call="web.html_container">
+        <t t-if="registry_type == 'customer'">
+            <t t-set="title" t-value="'Registro Iva Vendite'"/>
+        </t>
+        <t t-if="registry_type == 'supplier'">
+            <t t-set="title" t-value="'Registro Iva Acquisti'"/>
+        </t>
+        <t t-if="registry_type == 'corrispettivi'">
+            <t t-set="title" t-value="'Registro Corrispettivi'"/>
```

Comment:
Queste mi sembra non siano traducibili, andrebbero comunque eliminate le maiuscole:
Registro IVA vendite
Registro IVA acquisti
Registro corrispettivi

Lo stesso per eventuali altre stringhe che si trovano in questo file.